admin/apple-actions/index/class-export.php

```diff
@@ -57,7 +57,7 @@ public function fetch_exporter() {
 		$post       = get_post( $this->id );
 
 		// Build the excerpt if required
-		$excerpt = ( empty( $post->post_excerpt ) ) ? wp_trim_excerpt( $post->post_content ) : $post->post_excerpt;
+		$excerpt = ( empty( $post->post_excerpt ) ) ? wp_trim_words( strip_shortcodes( $post->post_content, 55 ) ) : $post->post_excerpt;
```

Comment:
Sorry. Fixed that. 

I think there is already a unit test for [excerpt here](https://github.com/alleyinteractive/apple-news/blob/master/tests/apple-exporter/builders/test-class-metadata.php#L20-L27).